src/api/journalEntries/journalEntry.ts

```diff
@@ -31,10 +31,11 @@ export const useJournalEntry = (id: string) => {
 };
 
 const createJournalEntryMutation = gql`
-  mutation createJournalEntry($data: JournalEntryCreateInput!){
+  mutation createJournalEntry($data: JournalEntryCreateInputData!){
     createJournalEntry(
       data: $data
     ) {
+      id
```

Comment:
The id is now needed in the return because the return is used to set the cached data on success